src/Traits/ReviewPermission.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\ContentReview\Traits;
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+trait ReviewPermission
+{
+    /**
+     * Whether or not a user is allowed use content review
+     */
+    protected function canUseReviewContent(DataObject $record, ?Member $user = null): bool
```

Comment:
```suggestion
    protected function isContentReviewable(DataObject $record, ?Member $member = null): bool
```
`$member` is used throughout the codebase, `$user` is not

I'm finding `canUseReviewContent` to be a strangely named because it's very similar to `canReviewContent`, as well as because it's prefixed with `can` which normally lives on $records, while here it's a utility triat.  I think renaming this to `isContentReviewable` fixes this

Reply:
Agreed because canBeReviewedBy is already being called on the Review form we don't need to duplicate this check.